examples/nextjs/next.config.js

```diff
@@ -1,5 +1,9 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+  experimental: {
+    esmExternals: 'loose'
+  }
+}
```

Comment:
I pulled your branch, removed this part, and it still works 🤷‍♂️

Reply:
That's weird because it tries to use `core/dist/index.cjs.js` which doesn't import but requires `@simplewebauthn/browser` which is a pure ESM package. It doesn't work for me without `esmExternals` set to `loose`. Did you restart the process after changing the settings?

Reply:
I cleaned everything from `node_modules` to `.next`, reinstalled and built, no error without `esmExternals`.
That is weird indeed

Reply:
Okay, I found the issue. It was the difference between the dev and prod Vite configuration. When I was running `pnpm dev` in the root folder, it treated `@simplewebauthn/browser` as an external dependency, because the process was using the generic Vite dev configuration, not the one specified in `packages/core`.